src/encoded/tests/features/test_trackhubs.py

```diff
@@ -1,6 +1,10 @@
 import pytest
 
-pytestmark = [pytest.mark.indexing]
+
+pytestmark = [
+    pytest.mark.bdd,
```

Comment:
so the trackhubs test is now run with BDD?   Does it  use the same fixture?

Reply:
It's a different case since test_trackhubs is a child of features folder. It means the index_workbook is used as a proper pytest fixutre and not explicitly imported.